packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -671,6 +671,15 @@ class _FocusState extends State<Focus> {
     Widget child = widget.child;
     if (widget.includeSemantics) {
       child = Semantics(
+        // Automatically request the focus for a focusable widget when it
+        // receives an input focus action from the semantics. Nothing is needed
+        // for losing the focus because if focus is lost, that means another
+        // node will gain focus and take focus from this widget.
+        // TODO(gspencergoog): Allow this to be set on iOS once the issue is debugged.
+        // Issue: https://
```

Comment:
Here's a link to the issue: https://github.com/flutter/flutter/issues/150030

Reply:
Added